src/containers/NFT/styles.scss

```diff
@@ -0,0 +1,54 @@
+@import '../shared/css/variables.scss';
+
+.token-page {
+  width: 100%;
+  .disclaimer-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 0px 32px;
```

Comment:
people tend to prefer not to use units for a value of 0 in CSS: https://stackoverflow.com/questions/7923042/when-specifying-a-0-value-in-css-should-i-explicitly-mark-the-units-or-omit